### DISCUSSION TITLE
Changing favico in _config.yml throws liquid error

### DISCUSSION THREAD

**Discussion by FulminatingAlchemist** [alshedivat/al-folio#2637] [Q&A] [answered]
I wanted to change my favicon, by editing _config.yml but kept getting the following error.
` Liquid Exception: No such file or directory @ rb_sysopen - assets/img/fav.ico in /home/runner/work/fulminatingalchemist.github.io/fulminatingalchemist.github.io/_layouts/default.liquid`

**Comment by abhilesh** [comment, marked as answer]
Is your githup repo up-to-date? I can't find a fav.icon in the [assets/img/](https://github.com/FulminatingAlchemist/fulminatingalchemist.github.io/tree/master/assets/img) directory

**Reply by FulminatingAlchemist** [reply]
I  pushed it into the wrong branch.
Solved
